sunpy/spectra/sources/callisto.py

```diff
@@ -93,6 +93,11 @@ def query(start, end, instruments=None, url=DEFAULT_URL):
             opn.close()
         day += _DAY
 
+@deprecated('0.8', alternative='callisto.search')
+def query(start, end, instruments=None, url=DEFAULT_URL):
+    __doc__ = search.__doc__
+    return search(start, end, instruments=instruments, url=url)
```

Comment:
Probably I'll have to propagate this to sunpy/spectra, or move callisto to sources in unidown.